src/dropdown.option.ts

```diff
@@ -483,6 +482,28 @@ export default class DropdownOption extends LitElement {
     </div>`;
   }
 
+  override updated() {
+    // `this.ariaSelected` needs to be updated whenever `disabled`, `privateMulitple`,
```

Comment:
`updated()` is usually a code smell. But I believe this is an legitimate use for it.